hack/rebasebot-hook-scripts/generate-ocp-manifests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# This script will be run by rebasebot during the rebase of the CAPI repositories. The location is specified as a parameter. It is downloaded from this repository and executed on the repository being rebased.
```

Comment:
Can you add a comment saying that this expects there exists a make target in the repo being rebased, under the `openshift` folder? That actually performs the ocp manifests generation?
Thanks!

Reply:
Added and improved the comment a bit.